nixos/modules/services/networking/bore/server-options.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
+  config,
+  ...
+}:
+let
+  inherit (lib) mkOption mkEnableOption types;
+  inherit (types)
+    port
+    str
+    ;
+in
+{
+  imports = [
+    ./common-options.nix
+  ];
+
+  options = {
+    enable = mkEnableOption "Bore TCP tunnel (remote proxy server) for `<name>`";
+
+    min-port = mkOption {
+      type = port;
+      default = 1024;
+      description = ''
+        Minimum accepted TCP port number.
+      '';
+    };
+
+    max-port = mkOption {
+      type = port;
+      default = 65535;
+      description = ''
+        Maximum accepted TCP port number.
+      '';
+    };
+
+    bind-addr = mkOption {
+      type = str;
+      default = "0.0.0.0";
+      description = ''
+        IP address to bind to, clients must reach this.
+      '';
+    };
+
+    bind-tunnels = mkOption {
+      type = str;
+      default = config.bind-addr;
+      defaultText = "bind-addr";
```

This will render as:

> Default: `"bind-addr"`

Instead, you want something like:


```nix
      defaultText = lib.literalExpression "config.services.bore.servers.<name>.bind-addr";
```

Which would also make the note in the description redundant.


I believe you can also do something like:

```suggestion
      defaultText = lib.literalExpression "config.${options.bind-addr}";
```

To more reliably get the same thing.

When you interpolate an option (from `options.*`) into a string, it is stringified as the full option path, complete with any submodule prefixes.